src/plugins/foxPage/utils/getFoxPageRouteAssetId.ts

```diff
@@ -14,10 +10,18 @@ export const getFoxPageRouteAssetId = (pathname: string) => {
   })
 
   if (foxPageAssetIdPathMatch) {
-    const foxAsset = foxPageAssetIdPathMatch?.params?.foxAsset ?? FOX_PAGE_DEFAULT_ASSET
+    const foxAsset = (foxPageAssetIdPathMatch?.params?.foxAsset ?? FOX_PAGE_DEFAULT_ASSET) as
+      | 'fox'
+      | 'foxy'
```

Comment:
It might be nicer to set the generic type of `foxPageAssetIdPathMatch`.

```suggestion
    const foxPageAssetIdPathMatch = matchPath<{ foxAsset?: 'fox' | 'foxy' }>(pathname, {
    path: '/fox/:foxAsset?',
  })

  if (foxPageAssetIdPathMatch) {
    const foxAsset = foxPageAssetIdPathMatch?.params?.foxAsset ?? FOX_PAGE_DEFAULT_ASSET
```

Reply:
Nice one ser, that looks way cleaner than the sneaky cast!
Removed in c36d43dc3d65633f3d0c88c2e9cf17389be59763